.circleci/config.yml

```diff
@@ -14,16 +14,16 @@ jobs:
           command: |
             source venv/bin/activate
             pip install -r requirements.txt
-            pip install coverage
+            #pip install coverage
+            pip install coveralls
       - run:
           name: Run tests with coverage
           command: |
             source venv/bin/activate
             coverage run setup.py test
-            coverage xml -i
       - run:
-          name: Send coverage results to CodeCov
-          command: bash <(curl -s https://codecov.io/bash)
+          name: Send coverage results to Coveralls
+          command: coveralls
```

Comment:
I would pull from `master` to get a small fix I made. Second part, is, you need to source venv/bin/activate here to be able to run this